sources/Inputs/include/NetworkManager.h

```diff
@@ -114,17 +106,7 @@ class NetworkManager {
    *
```

Comment:
Is that only generators that regulate those bus now or generators and/or VSC converters? If the latter, as it seems to me, consider re-wording

Reply:
solved